deployment/centos-package-x64/Dockerfile

```diff
@@ -20,6 +20,10 @@ RUN yum update -y \
  && mkdir -p ${SOURCE_DIR}/SOURCES \
  && ln -s ${PLATFORM_DIR}/pkg-src ${SOURCE_DIR}/SOURCES
 
+# Install yarn package manager
+RUN wget -q -O- https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo \
+ && yum install -y yarn
```

Comment:
don't you need to `yum update` after adding a repo?

Reply:
Nope, the update happens during the install.